src/pytest_plugins/consume/hive_simulators/conftest.py

```diff
@@ -28,12 +28,54 @@ def eth_rpc(client: Client) -> EthRPC:
     return EthRPC(ip=client.ip)
 
 
+@pytest.fixture(scope="function")
+def hive_consume_command(
+    test_suite_name: str,
+    client_type: ClientType,
+    test_case: TestCaseIndexFile | TestCaseStream,
+) -> str:
+    """
+    Command to run the test within hive.
+    """
+    return (
+        f"./hive --sim ethereum/{test_suite_name} "
+        f"--client-file configs/develop.yaml "
+        f"--client {client_type.name} "
+        f'--sim.limit "{test_case.id}" '
+        f"--docker.output "
```

Comment:
I almost never use this one, is it necessary?

Reply:
You mean `--docker.output` or the entire command?

I tend to use the former *almost* all the time, and check the logs via the terminal over instead of the `workspace/`.